edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/controllers/DeviceScopeController.cs

```diff
@@ -291,9 +324,39 @@ public static void UnauthorizedActor(string actorDeviceId, string actorModuleId,
                 Log.LogError((int)EventIds.UnauthorizedActor, $"{actorDeviceId}/{actorModuleId} not authorized to act OnBehalfOf {targetDeviceId}");
             }
 
-            public static void InvalidAuthchain(string authChain)
+            public static void InvalidRequestAuthchain(string authChain)
+            {
+                Log.LogError((int)EventIds.InvalidRequestAuthchain, $"Invalid auth chain: {authChain}");
+            }
+
+            public static void AuthFail_BadActor(string actorDeviceId, string actorModuleId)
+            {
+                Log.LogError((int)EventIds.AuthFail_BadActor, $"Only EdgeHub is allowed to act OnBehalfOf another identity: {actorDeviceId}/{actorModuleId}");
```

Comment:
Nit: suggest reword - 
{actorDeviceId}/{actorModuleId} is not authorized to connect on behalf of {targetDeviceId}